setup.py

```diff
@@ -50,6 +50,7 @@ def check_setuptools_features():
     'pytest>=3.0.0',
     'pytest-catchlog>=1.2.2',
     'pytest-cov>=2.2.1',
+    'pytest-mock',
```

Comment:
It's a thin wrapper which makes it very convenient to use `mock` with pytest. See https://github.com/pytest-dev/pytest-mock